tests/integration/pilot/locality/prioritized_test.go

```diff
@@ -118,14 +118,15 @@ func TestPrioritized(t *testing.T) {
 						BuildOrFail(ctx)
 
 					fakeHostname := fmt.Sprintf("fake-eds-external-service-%v.com", r.Int())
+
 					deploy(ctx, ns, serviceConfig{
 						Name:             "prioritized-eds",
 						Host:             fakeHostname,
 						Namespace:        ns.Name(),
 						Resolution:       "STATIC",
-						ServiceBAddress:  b.WorkloadsOrFail(ctx)[0].Address(),
+						ServiceBAddress:  b.Address(),
 						ServiceBLocality: "region/zone/subzone",
-						ServiceCAddress:  c.WorkloadsOrFail(ctx)[0].Address(),
+						ServiceCAddress:  c.Address(),
```

Comment:
@howardjohn any idea why the Pod address was directly used instead of the service IP ?

Reply:
Not sure, this was one of the first tests on the new framework so maybe the `b.Address()` function didn't exist?

Reply:
👍 okay, I only wanted to cross-check that there is no reason to not use the service IP